### PR TITLE
Add a "Write your first test" page to the Get Started section

### PR DESCRIPTION
**Pull request by oleiade**
## What?

As we agreed during our latest meetup, this PR adds a "Write your first test" page to the Getting Started section.

It aims to cover a gap between installing k6, and "Running k6", as we realised we had not introduced users to scripting in the logical flow yet. This PR aims to bridge that gap, and links to a lot of useful resources users might find useful on their onboarding experience with k6.

@heitortsergent this is really a draft on my side, you should feel free to propose whatever you see fit 👍🏻 Also, I've noticed we have some pages in our documentation which might repeat this information, we might want to also reorganize/regroup it somehow 🙇🏻 

## Checklist

- [X] I have used a meaningful title for the PR.
- [X] I have described the changes I've made in the "What?" section above.
- [X] I have performed a self-review of my changes.
- [X] I have run the `npm start` command locally and verified that the changes look good.
- [X] I have made my changes in the `docs/sources/next` folder of the documentation.

### CODE REVIEW

**Comment by oleiade** [comment]
🏓 @heitortsergent I've addressed some of the feedback you offered last time we spoke 🙇🏻

File: docs/sources/next/get-started/write-your-first-test.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+1. _Multiple requests_: You can add more `http.get()` or `http.post()` requests to simulate more complex user flows.
+2. _Using TypeScript_: If you prefer TypeScript, k6 supports it too. You can learn more in our [TypeScript guide](https://grafana.com/docs/k6/<K6_VERSION>/using-k6/javascript-typescript-compatibility-mode/#experimental-enhanced-mode).
+3. _Thresholds, checks, and metrics_: You can add conditions to monitor performance. For example, you can set thresholds to ensure the response time doesn’t exceed a certain limit. Learn more about thresholds and checks in our documentation.
+4. _Browser tests_: Use the browser module to simulate user interactions like clicking buttons or filling out forms. This is useful for testing web applications. Learn more about browser tests in our [documentation](https://grafana.com/docs/k6/<K6_VERSION>/using-k6-browser/).
+
+Furthermore, to speed up the process of putting together a k6 test script in the future, you can use the k6 to generate a basic script like the one we've written above using the `k6 new` command. Try it out!
```
Comment (heitortsergent) [inline docs/sources/next/get-started/write-your-first-test.md:93]:
```suggestion
You can also use the `k6 new` command to speed up the process of putting together a k6 test script when you're testing a new service or application. Try it out!
```

File: docs/sources/next/get-started/write-your-first-test.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## Next Steps
```
Comment (heitortsergent) [inline docs/sources/next/get-started/write-your-first-test.md:95]:
```suggestion
## Next steps
```

File: docs/sources/next/get-started/write-your-first-test.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+Now that you’ve written your first k6 test script, it’s time to run it. Visit our [Running k6](https://grafana.com/docs/k6/<K6_VERSION>/get-started/running-k6/) page to learn how to execute your script and analyze the results.
```
Comment (heitortsergent) [inline docs/sources/next/get-started/write-your-first-test.md:97]:
```suggestion
Now that you’ve written your first k6 test script, it’s time to run it. Refer to [Running k6](https://grafana.com/docs/k6/<K6_VERSION>/get-started/running-k6/) to learn how to execute your script and analyze the results.
```

File: docs/sources/next/get-started/write-your-first-test.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+   export default function () {
+     // Make a GET request to the target URL
+     http.get('https://test-api.k6.io');
+
+     // Sleep for 1 second to simulate real-world usage
+     sleep(1);
+   }
+   ```
+
+### Going a bit further
```
[the remark this answers is not in the document]
Comment (oleiade) [reply docs/sources/next/get-started/write-your-first-test.md:84]: I like it 👍🏻

File: docs/sources/next/get-started/write-your-first-test.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+2. **Imports**: You can import additional [k6 modules](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/) or [JavaScript libraries (jslibs)](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/jslib/) to extend your script’s functionality, such as making HTTP requests or simulating browser interactions. Note that k6 is not built upon NodeJS, and instead uses its own JavaScript runtime. Compatibility with some npm modules may vary.
+3. **Options (optional)**: Enable you to configure the execution of the test, such as defining the number of virtual users, the test duration, or setting performance thresholds. See our [options documentation](https://grafana.com/docs/k6/<K6_VERSION>/using-k6/k6-options/) for more details.
+4. **Lifecycle operations (optional)**: Because your test might need run code before and/or after the execution of the test logic, [lifecycle operations](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/jslib/) allow you to write code, either as predefined functions, or within specific code scopes, that will be executed at different stages of the test execution.
```
[the remark this answers is not in the document]
Comment (oleiade) [reply docs/sources/next/get-started/write-your-first-test.md]: I rephrased accordingly 🙂 Let me know what you think 🙇‍♂️

**Review by heitortsergent — APPROVED** [review]
Thanks for making the changes @oleiade. 🙇

**Pull request merged**